jujucrashdump/crashdump.py

```diff
@@ -48,6 +48,8 @@
     '/var/lib/libvirt/filesystems/plumgrid-data/log',
     '/var/lib/libvirt/filesystems/plumgrid/var/log',
     '/var/log',
+    # /root/curtin-install-cfg.yaml /root/curtin-install.log
+    '/root',
```

Comment:
Do we actually want to pull everything from that directory?  Seems like there might be sensitive files there, depending on the charm.  For example, the Kubernetes charms [write certs and their private keys](https://github.com/charmed-kubernetes/layer-kubernetes-common/blob/master/lib/charms/layer/kubernetes_common.py#L39) there.